### PR TITLE
Adding name to test cases in encoding_test.go

### PR DESCRIPTION
**Pull request by surajnarwade**
Resolves second task from issue #99

### CODE REVIEW

**Comment by surajnarwade** [comment]
cc @containscafeine

File: pkg/encoding/v1/encoding_test.go
Comment (concaf) [inline pkg/encoding/v1/encoding_test.go]: Instead of using `case-1`, `case-2`, etc, can we put the `RawPort` in there? Like `Failed Portmapping - x5000`? WDYT?

File: pkg/encoding/v1/encoding_test.go
Comment (concaf) [inline pkg/encoding/v1/encoding_test.go]: How about using `t.Run(tt.name, func(t *testing.T)`, you can find the references in other test cases in the same file.

File: pkg/encoding/v1/encoding_test.go
Comment (concaf) [inline pkg/encoding/v1/encoding_test.go]: same as above

File: pkg/encoding/v1/encoding_test.go
Comment (concaf) [inline pkg/encoding/v1/encoding_test.go]: same as above, `t.Run....`

File: pkg/encoding/v1/encoding_test.go
Comment (concaf) [inline pkg/encoding/v1/encoding_test.go]: Let's put the `RawEnvVar` value in the test name?

File: pkg/encoding/v1/encoding_test.go
Comment (concaf) [inline pkg/encoding/v1/encoding_test.go]: `t.Run...`

File: pkg/encoding/v1/encoding_test.go
Comment (concaf) [inline pkg/encoding/v1/encoding_test.go:586]: `t.Run...`

**Comment by surajnarwade** [comment]
cc @surajssd @containscafeine

File: pkg/encoding/v1/encoding_test.go
Comment (concaf) [inline pkg/encoding/v1/encoding_test.go]: Should we add `t.Run(test.Name, func(t *testing.T)` besides/instead of `t.Log...` here? WDYT?

**Review by concaf — COMMENTED** [review]
@surajnarwade looks good overall.
So, in the test cases where we use `t.Run` to run the tests, do we require `t.Log`? Instead, we can add the name like `t.Run(test.Name, func(t *testing.T)`, where currently it is, `t.Run(" ", func(t *testing.T)`

At some time, we should probably make all tests run using `t.Run`, but not in this PR.

@surajssd WDYT?

File: pkg/encoding/v1/encoding_test.go
Comment (surajnarwade) [reply pkg/encoding/v1/encoding_test.go]: @containscafeine , `t.Run` don't print test case name so I put `t.log`

**Comment by concaf** [comment]
@surajnarwade LGTM, let's merge this once @surajssd gives a :+1:
Thanks for taking this up! :)

**Review by concaf — APPROVED** [review]

**Comment by pradeepto** [comment]
@surajssd @containscafeine @surajnarwade Why is this blocked for 22 days? This should be handled on a priority basis. P0.

**Comment by kadel** [comment]
This needs to be rebased. ping @surajnarwade 

@pradeepto 
> This should be handled on a priority basis. P0.

Why P0?

**Comment by kadel** [comment]
tests are failing

**Comment by pradeepto** [comment]
@kadel Its been pending after a LGTM for a long time. Needed the second LGTM which fell trhough the cracks. Now I am sure this issue needs a rebase and probably will fix the issues.

**Comment by surajnarwade** [comment]
@kadel , env defintions are refactored, thats why tests were failing, now I changed tests accordingly

**Comment by kadel** [comment]
@surajnarwade thank you ;-)

**Pull request merged**
